examples/nemo/scripts/omp_gpu_trans.py

```diff
@@ -108,11 +116,7 @@ def trans(psy):
                 region_directive_trans=omp_target_trans,
                 loop_directive_trans=omp_loop_trans,
                 # Collapse is necessary to give GPUs enough parallel items
-                collapse=True,
-                # We exclude loops with calls when parallelising, with the
```

Comment:
Doing it in file-granularity was problematic because the ecmwf has a call which shouldn't be included in lib_fortran. The other optimizations make up for this change. But I think this could be re-introduced by analysing or listing the permitted calls (see new code comment above).